src/openforms/formio/rendering/tests/test_vanilla_formio_components.py

```diff
@@ -1236,3 +1236,51 @@ def test_fieldset_with_logic_depending_on_selectboxes(self):
         # Check that the fieldset is present
         # Nodes: Form, SubmissionStep, Fieldset, Component (textfield), Component (radio)
         self.assertEqual(len(nodes), 5)
+
+    def test_partners(self):
```

Comment:
this is the wrong test file - it's meant for built-in formio component types (vanilla in the name + see the docstring at the beginning of the test file). please put the tests for custom components in a different file